### PR TITLE
Fix PKGBUILD and update it to latest release

### PR DESCRIPTION
**Pull request by Itai-Nelken**
Signed-off-by: Itai-Nelken <itainelken@gmail.com>

### CODE REVIEW

**Comment by CleanMachine1** [comment]
The AUR already has it up to date, so I can either make it identical to this or vice versa 

https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=dollarskip

File: PKGBUILD [hunk 1]
```diff
@@ -18,4 +18,4 @@ package() {
     install -m 755 -D temp "${pkgdir}/usr/local/bin/\$"
```
Comment (CleanMachine1) [inline PKGBUILD:18]:
```suggestion
    install -m 755 -D dollarskip "${pkgdir}/usr/local/bin/\$" 
```

Comment (Itai-Nelken) [reply PKGBUILD]: I meant to change that with the features of the new makefile. See latest commit.

File: PKGBUILD [hunk 2]
```diff
@@ -1,7 +1,7 @@
 pkgname="dollarskip"
-pkgver="3.1.2"
+pkgver="3.1.3"
 pkgrel="1"
-source=(https://github.com/CleanMachine1/DollarSkip/archive/3.1.2.tar.gz)
+source=(https://github.com/CleanMachine1/DollarSkip/archive/refs/tags/v3.1.3.tar.gz)
```
Comment (CleanMachine1) [inline PKGBUILD:4]: This suggested link or https://github.com/CleanMachine1/DollarSkip/archive/v3.1.3.tar.gz

the 2nd is more succinct

File: PKGBUILD [hunk 3]
```diff
@@ -14,8 +14,8 @@ build() {
 }
 
 package() {
-    cd DollarSkip-${pkgver} &&
-    install -m 755 -D temp "${pkgdir}/usr/local/bin/\$" 
+    cd DollarSkip-${pkgver}
+    make install DESTDIR="${pkgdir}"
```
Comment (CleanMachine1) [inline PKGBUILD:18]: The AUR will need to be different, since this installs to /usr/local/bin

Comment (Itai-Nelken) [reply PKGBUILD:18]: So we can add `PREFIX="/usr"` to the makefile for the aur.

File: PKGBUILD (hunk 2 above)
Comment (CleanMachine1) [reply PKGBUILD:4]: Please apply my link I have suggested unless you have anything to add.

```suggestion
source=(https://github.com/CleanMachine1/DollarSkip/archive/v3.1.3.tar.gz)
```

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**